pkgs/by-name/ma/marytts/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  # Gradle 8 complains about implicit task dependencies when using `installDist`.
+  # See marytts/marytts#1112
+  gradle_7,
+  makeWrapper,
+  jdk,
+  nixosTests,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "marytts";
+  version = "5.2.1";
+
+  src = fetchFromGitHub {
+    owner = "marytts";
+    repo = "marytts";
+    rev = "1c2aaa0751b7cef8ae83216dd78b4c61232b3840";
```

Comment:
Please fetch the tag

https://github.com/marytts/marytts/tree/v5.2.1

Reply:
I mentioned this above in the PR description as 5.2.1 is quite literally impossible to package (you're welcome to try if you want to)
> NOTE: The latest stable version, 5.2.1, is unfortunately impossible to package as there exists no prebuilt artifacts of it remaining and some of its dependencies have been pulled from Maven Central. The latest unstable version works just fine though